src/azure-cli/azure/cli/command_modules/resource/commands.py

```diff
@@ -253,6 +253,7 @@ def load_command_table(self, _):
         g.custom_command('invoke-action', 'invoke_resource_action', transform=DeploymentOutputLongRunningOperation(self.cli_ctx))
         g.generic_update_command('update', getter_name='show_resource', setter_name='update_resource',
                                  client_factory=None)
+        g.custom_command('patch', 'patch_update_resource')
```

Comment:
It is suggested to change the method name `patch_update_resource` to `patch_resource`, because `update` and `patch` are verbs, but there is usually only one verb in the method name.